lib/authorization.rb

```diff
@@ -51,7 +51,10 @@ def self.generate_default_acls(subject, object)
 
   module Dataset
     def authorized(subject_id, actions)
-      where(id: Authorization.authorized_resources(subject_id, actions))
+      # We can't use "id" column directly, because it's ambiguous in big joined queries.
+      # We need to determine table of id explicitly.
+      from = @opts[:from].first
```

Comment:
sneaky, where does `@opts` come from? I imagine it's make-sense thing in sequel, but help out the reader (me) a bit if it's short enough (if it's not, a link is better)

Reply:
I added comment